README.md

```diff
@@ -58,6 +59,7 @@
 - 💵 [Infinite Studio](https://www.infinitestudio.art) *(Android, iOS)*
 - 💵 [Realistic Paint Studio](https://realisticpaint.com) *(Windows, Mac, iOS)*
 - 💵 [ArtRage](https://www.artrage.com/artrage) *(Windows, Mac, Android, iOS)*
+- ✨ [Tux Paint] (https://tuxpaint.org/) *(Windows, Mac, Linux)*
```

Comment:
Free and open-source are always on top of lists according to the paragraph at head of the README